tests/test_pipeline_mixin.py

```diff
@@ -236,67 +324,82 @@ def run_pipeline_test(
                 A subclass of `PretrainedModel` or `PretrainedModel`.
             tokenizer_name (`str`):
                 The name of a subclass of `PreTrainedTokenizerFast` or `PreTrainedTokenizer`.
-            processor_name (`str`):
+            image_processor_or_feature_extractor_name (`str`):
```

Comment:
This looks like not the correct docstring?

Reply:
addressed in https://github.com/huggingface/transformers/pull/32514/commits/403cd0b810413da5e462f28410c224e9d8eca39d